### DISCUSSION TITLE
Where is PNG Info?

### DISCUSSION THREAD

**Discussion by hollowstrawberry** [vladmandic/sdnext#301] [Q&A] [answered]
🤔

**Comment by vladmandic** [comment]
same question for 20+ times. normally i answer, but this time i'll say please search for issues/discussions. hint: yes, it exists.

**Reply by hollowstrawberry** [reply, marked as answer]
> same question for 20+ times. normally i answer, but this time i'll say please search for issues/discussions. hint: yes, it exists.

Oh my bad. I searched yesterday and there were no results for "png info", I searched now and there's 1 result from 4 hours ago.

You're supposed to upload it into the Process Image tab, after which you get the png info automatically.

The functionality is very unclear, which is probably why you get the same question 20+ times. If this fork goes mainstream it might be worth considering how to make this clearer or even reverting to the old way.

**Comment by vladmandic** [comment]
> The functionality is very unclear, which is probably why you get the same question 20+ times. If this fork goes mainstream it might be worth considering how to make this clearer or even reverting to the old way.

I'd disagree. I think its more intuitive where it is now, its just that people are used to A1111 and don't look elsewhere. If everything was to remain the same, what would be the point of me doing development? :)

**Reply by hollowstrawberry** [reply]
I agree conceptually, it's just that I looked in the "Process Image" tab too and didn't realize you could get PNG Info from there. Plus, reading existing data is not what I'd call "processing". But that's just my 2 cents. Thank you for the work.

**Reply by gruevy** [reply]
It's not more intuitive. It's less intuitive and less useful. Why is it 'parameters' and 'full metadata' instead of something readable like positive and negative prompts? And where's the 'send to txt2img' button? That's the functionality I really miss. That's something I actually do pretty often, bring up old images to play around with variations on prompts or new models. I used to be able to drag+drop, click, go, and now you're making me copy and paste 1 field into 2 fields on separate tabs and it's worse.

**Reply by vladmandic** [reply]
Button is right there immediately below image as soon as you load it.
Take a minute to look instead of venting, sorry.
And full metadata is any metadata that can be in image. Not just params from WebUI.

**Reply by gruevy** [reply]
If you mean this button, the only thing it does is crash the UI and I have to reload the page. None of the other buttons I see on that page look like they're supposed to send generation into to the 'from text' tab. If it's a different one, maybe it's not loading for me? What one is it?
![image](https://user-images.githubusercontent.com/1585698/233541421-7d4eee82-9cdc-46e9-81a7-b7327a3743b1.png)

**Reply by gruevy** [reply]
LMK if that was the button and you'd like a bug report, happy to oblige

**Reply by vladmandic** [reply]
Yes, that's the button - and yes, create an issue for that.
Just make sure you're on latest as there were already some fixes regarding that this afternoon.

**Reply by gruevy** [reply]
Sweet, looks like that fixed it. Working now, I guess I missed the update this morning when I installed. Thanks!
